gzipcompressor/gzipcompressor.go

```diff
@@ -34,3 +36,20 @@ func CompressString(str string) (string, error) {
 	bytes := []byte(str)
 	return CompressData(bytes)
 }
+
+// DecompressString given a gzipped and base64 encoded string will decode back to
+// uncompressed text
+func DecompressString(d string) (string, error) {
```

Comment:
Renaming `d` var to something like `encodedString` will make it easier to follow and the comment no longer necessary.

Reply:
Yes, I see your point that the code can be self-documenting but are we interested in documenting our packages with godoc?  My IDE (vscode) spends a lot of time informing me that I should comment any function/method that is exported.  What do you think?

Reply:
I prefer to avoid comments and make things explicit but just my preference